core/assets/bundles/bundle_fr.properties

```
@@ -1084,7 +1088,7 @@ mech.alpha-mech.description = Le mécha standard. Est basé sur une unité Poign
 mech.delta-mech.description = Un mécha rapide, avec une armure légère, conçu pour les attaques de frappe. Il inflige, par contre, peu de dégâts aux structures. Néanmoins il peut tuer de grand groupes d'ennemis très rapidement avec ses arcs électriques.
 mech.tau-mech.description = Un mécha de support. Soigne les blocs alliés en tirant dessus. Il peut aussi éteindre les feux et soigner ses alliés en zone avec sa compétence.
 mech.omega-mech.description = Un mécha cuirassé et large fait pour les assauts frontaux. Sa compétence lui permet de bloquer 90% des dégâts.
-mech.dart-ship.description = Le vaisseau standard. Raisonnablement rapide et léger. Il a néanmoins peu d'attaque et une faible vitesse de minage.
+mech.dart-ship.description = Le vaisseau standard. Raisonnablement rapide et léger, et possède une vitesse de minage rapide. Néanmoins, ses capacitées d'attaque sont faibles.
```

`et` twice is not allowed. `possède` must have a subject.
```suggestion
mech.dart-ship.description = Le vaisseau standard. Il est raisonnablement rapide, léger et possède une vitesse de minage rapide. Néanmoins, ses capacitées d'attaque sont faibles.
```